Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:14-alpine
```

Comment:
The answer to this question is probably tied up in #50, but is it okay if this drifts from the version specified in our `package.json` (currently 13)? I have no particular Node version allegiance, but a slight preference for LTS > current (unless we have specific features we need that aren't in LTS). Mostly just curious if we need to ensure this is in sync with the package file.

Reply:
`package.json` specifies a minimum, so I think exceeding it is okay. But Travis is also set to test on 13, so there are a couple signals 13 is the current default. It's also EOL now, which we mostly shouldn't care about, except that we want the Docker image and everything else to have the longest functional life possible.